nx-X11/programs/Xserver/Imakefile

```diff
@@ -427,7 +427,7 @@ $(NXAGENTOBJS) $(NXAGENTLIBS) $(NXAGENTSYSLIBS):: $(NXAGENTDIRS)
 #if defined(SunArchitecture)
 NXAGENTNXLIBS = -L ../../../nxcomp -L ../../../nxcompext -L ../../../nxcompshad \
                 -lXcomp -lXcompext -lXcompshad -lrt -L/usr/sfw/lib -lNX_Xrender -lNX_Xfixes -lXfont \
-                -L../../../nx-X11/exports/lib -lNX_Xtst -lNX_Xdamage -lNX_Xrandr -lNX_Xcomposite -lXdmcp \
```

Comment:
On  Di 12 Mai 2015 05:49:40 CEST, Mihai Moldovan wrote:

>> @@ -427,7 +427,7 @@ $(NXAGENTOBJS) $(NXAGENTLIBS)  
>> $(NXAGENTSYSLIBS):: $(NXAGENTDIRS)
>>  #if defined(SunArchitecture)
>>  NXAGENTNXLIBS = -L ../../../nxcomp -L ../../../nxcompext -L  
>> ../../../nxcompshad \
>>                  -lXcomp -lXcompext -lXcompshad -lrt -L/usr/sfw/lib  
>> -lNX_Xrender -lNX_Xfixes -lXfont \
>> -                -L../../../nx-X11/exports/lib -lNX_Xtst  
>> -lNX_Xdamage -lNX_Xrandr -lNX_Xcomposite -lXdmcp \
>
> Removing... or replacing with `-lXdamage`?

Removing. nxagent has been overlinked with NX_Xdamage, so far. Xdamage  
linking is only needed in libXcompshad.so.3.
-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
fon: +49 (1520) 1976 148

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/freebusy/m.gabriel%40das-netzwerkteam.de.xfb